packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -472,6 +473,19 @@ class ListTile extends StatelessWidget {
   /// property.
   final Widget? trailing;
 
+  /// {@template flutter.material.inkwell.statesController}
+  /// Represents the interactive "state" of this widget in terms of
+  /// a set of [WidgetState]s, like [WidgetState.pressed] and
+  /// [WidgetState.focused].
+  ///
+  /// Classes based on this one can provide their own
+  /// [WidgetStatesController] to which they've added listeners.
+  /// They can also update the controller's [WidgetStatesController.value]
+  /// however, this may only be done when it's safe to call
+  /// [State.setState], like in an event handler.
+  /// {@endtemplate}
```

Comment:
To copy that documentation, you'd just have to do this
```suggestion
/// {@macro flutter.material.inkwell.statesController}
```